src/components/Badge/Badge.visual.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { Badge } from './';
+import {VisualContainerElement} from "../../../test/visual/VisualContainerElement";
+
+class BadgeVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
+        <VisualContainerElement>
+          <Badge {...this.props} />
+        </VisualContainerElement>
+      </TPAComponentsProvider>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
```

Comment:
Let's add the different priorities for the visual tests, so we know that the different styles won't get broken